migrate/20251209_add_pg_init_script.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:postgres_resource) do
+      add_column :init_script, :text, null: true
```

Comment:
What’s the limit for this column? Since we say `This is useful for setting up initial schemas, tables, or data`, I fear that some customer might try to put a large dataset into it.

Reply:
Makes sense. Added 2000 character limit, like we do in vm init scripts.